src/user.service.ts

```diff
@@ -0,0 +1,42 @@
+import{AxiosInstance} from "axios";
+import{CreateUserWithRole, User, LocationUser, FolderUser} from "./types/User";
+
+const endpointRoles = "roles";
+const endpointUsers = "users";
+
+export default (axios: AxiosInstance) => {
+    return{
+        getAllUsersRoles: (params: User) =>{
```

Comment:
this could be changed to getAllRoles, the reason of this is to implement Users.getAllRoles instead of User.getAllUsersRoles, in the current name we are repeating the word users in the same context